templates/todo/projects/csharp-cosmos-sql/.repo/bicep/repo.yaml

```diff
@@ -153,6 +153,10 @@ repo:
         - obj
         - .vs
 
+    # Auth C# common
+    - from: ../../../../common/auth/csharp
```

Comment:
I noticed we are adding the credential in `/todo/common` so it is propagated to all todo-templates.
Should we use the `common` folder from one level up?  so it is known to be applicable to all kinds of templates?

Reply:
Great idea.

Reply:
Yes, but lets do that after we get all the credentials in since right now I'm just adding this to the same area that was originally started.